Makefile

```diff
@@ -169,7 +179,7 @@ TMP_DIR=$$(mktemp -d) ;\
 cd $$TMP_DIR ;\
 go mod init tmp ;\
 echo "Downloading $(2)" ;\
-GOBIN=$(PROJECT_DIR)/bin go get $(2) ;\
+GOBIN=$(PROJECT_DIR)/bin go install $(2) ;\
```

Comment:
From my understanding `go install` will only install locally downloaded packages, `go get` will download and then run `go install`. I believe we should keep it as it is.

Reply:
no longer works under 1.18: https://go.dev/doc/go-get-install-deprecation

Reply:
> In Go 1.18, go get will no longer build packages; it will only be used to add, update, or remove dependencies in go.mod. Specifically, go get will always act as if the -d flag were enabled.

Should we be doing both `go get | go install`? Those packages will still need to be downloaded, right?

Reply:
Perfect! thnx